composer.json

```diff
@@ -7,29 +7,29 @@
         "php": ">=8.2",
         "spryker/event": "^2.4.0",
         "spryker/event-dispatcher-extension": "^1.0.0",
-        "spryker/kernel": "^3.49.0",
+        "spryker/kernel": "^3.72.0",
         "spryker/laminas": "^1.0.0",
-        "spryker/propel": "^3.34.0",
+        "spryker/propel": "^3.45.0",
         "spryker/propel-orm": "^1.16.0",
         "spryker/symfony": "^3.1.0",
         "spryker/util-encoding": "^2.1.0"
     },
     "require-dev": {
-        "codeception/module-asserts": "^1.3.0",
+        "codeception/module-asserts": "^3.0.0",
         "phpunit/phpunit": "^9.0.0",
         "spryker/application": "*",
-        "spryker/code-sniffer": "@stable",
+        "spryker/code-sniffer": "^0.17.1",
         "spryker/config": "^3.5.0",
         "spryker/console": "^3.2.0 || ^4.0.0",
-        "spryker/container": "^1.4.0",
+        "spryker/container": "^1.4.6",
         "spryker/error-handler": "*",
         "spryker/event-dispatcher": "^1.0.0",
         "spryker/log": "*",
         "spryker/monolog": "*",
         "spryker/queue": "*",
         "spryker/silex": "^2.0.0",
-        "spryker/testify": "^3.42.0",
-        "spryker/transfer": "^3.25.0"
+        "spryker/testify": "^3.58.0",
+        "spryker/transfer": "^3.39.0"
```

Comment:
same here. I don't see how improved array validation is relevant for the changes in the code

Reply:
the same